rust/gui-client/src-tauri/src/client.rs

```diff
@@ -157,7 +157,12 @@ fn fix_log_filter(settings: &mut AdvancedSettings) -> Result<()> {
 /// Don't drop the log handle or logging will stop.
 fn start_logging(directives: &str) -> Result<logging::Handles> {
     let logging_handles = logging::setup(directives)?;
-    tracing::info!(?GIT_VERSION, "`gui-client` started logging");
+    tracing::info!(
+        ?directives,
+        ?GIT_VERSION,
+        system_uptime_seconds = firezone_headless_client::uptime::get().map(|dur| dur.as_secs()),
```

Comment:
A `Duration` actually has a human-friendly debug output like "15m".

```suggestion
        ?system_uptime = firezone_headless_client::uptime::get(),
```

Reply:
For that I get 
```
the trait `tracing::Value` is not implemented for `std::time::Duration`
```

Reply:
Whoops, the `?` needs to go in the rvalue haha:

```
system_uptime = ?...
```